src/containers/Authorization/types.ts

```diff
@@ -0,0 +1,10 @@
+import { SwitchTabConst } from './index';
+
+export type IOnRegisterAction = (
+  userName: string,
```

Comment:
create an object with all field and use it everywhere, where you use these parameters